### DISCUSSION TITLE
V 2.0 beta question

### DISCUSSION THREAD

**Discussion by jlrosssc** [jeroenterheerdt/HAsmartirrigation#239] [General]
I installed the original version recently and was getting familiar with how it worked before creating automations with the sensor.  Then I saw the V2 beta with the awesome feature of multiple zones.  I installed it using the same input sensors I had on the original version.  These primarily come from my home weather station with a few sensors like barometric pressure coming from other sources.  I'm not clear that this is working, however.  I have 2 zones set up but show no apparent changes after 2 days of hot, dry weather when monitoring sensor.default_zone_1 and _2.  Is this where the output should show?  When I select the i icon for each zone, it says none, which I'm interpreting as no calculations have been made.  Am I looking the in the wrong place for the output?

**Comment by jeroenterheerdt** [comment]
Have you wired up the sensor groups? Please see readme for v2.0 and the wiki. If you need more help please share screenshots and diagnostic file in an issue

**Comment by jlrosssc** [comment]
It looks to have the sensor groups set up correctly.  Attached are a few screen shots
<img width="657" alt="Screenshot 2023-09-05 at 9 19 50 PM" src="https://github.com/jeroenterheerdt/HAsmartirrigation/assets/132500156/a4858f52-d0ac-4692-a16a-df46c90077be">
<img width="728" alt="Screenshot 2023-09-05 at 9 20 15 PM" src="https://github.com/jeroenterheerdt/HAsmartirrigation/assets/132500156/1d50caae-e6a3-4278-bb56-c4de2676645a">
<img width="692" alt="Screenshot 2023-09-05 at 9 20 44 PM" src="https://github.com/jeroenterheerdt/HAsmartirrigation/assets/132500156/96104af2-2079-4948-8441-0f25aaf4c985">

**Reply by jeroenterheerdt** [reply]
so the first update happens at 20:45  your time (8:45 PM), so have you waited until that has happened? You can also press the update button for a zone to trigger a manual update or pressure update all.

**Comment by jlrosssc** [comment]
I have adjusted the trigger time and hit the manual update with no change. Sent from my iPhoneOn Sep 5, 2023, at 9:41 PM, Jeroen ter Heerdt ***@***.***> wrote:﻿
so the first update happens at 20:45  your time (8:45 PM), so have you waited until that has happened? You can also press the update button for a zone to trigger a manual update or pressure update all.

**Reply by jeroenterheerdt** [reply]
Please open an issue using the template so we can help. I need a diagnostics file to be able to help, see issue template.
